src/main/java/com/fasterxml/jackson/databind/ser/std/BeanSerializerBase.java

```diff
@@ -401,6 +401,14 @@ public void resolve(SerializerProvider provider)
             prop.assignSerializer(ser);
         }
 
+        // also, any-getter may need to be resolved
+        for (int i = 0; i < _props.length; i++) {
+            BeanPropertyWriter prop = _props[i];
+            if (prop instanceof AnyGetterWriter) {
+                ((AnyGetterWriter) prop).resolve(provider);
```

Comment:
Why doe we need this, when there's resolution right after? Or conversely, is the resolve call below needed at all?

Reply:
Ah, thats Because I haven't managed to find a way to resolve case [here](https://github.com/FasterXML/jackson-databind/pull/4396/files#r1509905790), where field does not have public visibility.

Reply:
You are right, `_anyGetterWriter` is no longer needed. Removed `_anyGetterWriter` entirely 👍🏼